apps/web/app/(app)/gardens/[chain]/[garden]/page.tsx.orig

```diff
@@ -0,0 +1,273 @@
+<<<<<<< HEAD
+"use client";
+
+import { tree2, tree3, grassLarge } from "@/assets";
+import Image from "next/image";

```

Comment:
what is that file? merge backup?

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/b21d5df3735c22b65ad6db7ae0ff2b39ef75dae4